website/docs/cloud-docs/agents/request-forwarding.mdx

```diff
@@ -38,6 +38,7 @@ of accessing the target API.
 
 ## Agent Configuration
 
+### Enable request forwarding
```

Comment:
According to our style guide, we don't "stack headers" and it doesn't feel like we need another header here.
```suggestion
```